app/js/main.js

```diff
@@ -229,6 +229,9 @@ function startFlowAuto() {
       // Forget any subscriptions, for metering demo purposes.
       subscriptions.clear();
 
+      // Set up metering demo language.
+      document.documentElement.lang = getQueryParams().lang || 'en';
```

Comment:
haven't done language at google yet but in the past I've seen it always paired with a region string as well (e.g. en-US, es-US, en-GB, etc)

Reply:
The param can specify region string as well. It's not necessary for the default though